aya/src/maps/bloom_filter.rs

```diff
@@ -274,4 +276,11 @@ mod tests {
             Err(MapError::ElementNotFound)
         ));
     }
+
+    fn create_fd() -> OwnedFd {
+        let dir = env::temp_dir();
+        File::create(dir.join("f1"))
+            .expect("unable to create file in tmpdir")
+            .into()
+    }
```

Comment:
if we don't like this more manual approach we could bring in `tempfile` as a dev dependency or perhaps use memfds. This was just the easiest way to get the tests to pass for now.